tests/test_scripts/water_balance/combine_output.py

```diff
@@ -1,45 +1,29 @@
 #!/usr/bin/env python
-# type: ignore
-import os
+
 import re
-import sys
+from pathlib import Path
 
-import MF6_wbal_listing as readmf6
 import netCDF4 as nc
 import numpy as np
 
 # import xlwt, xlrd
-import openpyxl
 import pandas as pd
-import readfmhis as readdfm
-import readmsw as readmsw
-from xlutils.copy import copy as xl_copy
-
+from test_scripts.water_balance.MF6_wbal_listing import listfile2df
+from test_scripts.water_balance.readfmhis import hisfile2df
+from test_scripts.water_balance.readmsw import totfile2df  # type: ignore
```

Comment:
Can the type ignore annotation now be removed?

Reply:
no, because then mypy complains about the readmsw.py file. That uses the dateutil library, and that library does not have "stubs" according to mypy. I googled what that is, and it appears to be a file with metadata for mypy to be used with imported libraries. Without those, mypy gives errors when validating that file

Reply:
nice idea, done